data/types/base_test.go

```diff
@@ -972,6 +972,12 @@ var _ = Describe("Base", func() {
 			})
 		})
 
+		Context("GetTimeZoneOffset", func() {
+			It("gets the time zone offset", func() {
```

Comment:
there is never a chance we are always just comparing nil is nil as its never set?

Reply:
Yes, that is possible. This test is just to check that the function `GetTimeZoneOffset` returns the same value (`nil` or otherwise) as stored in the `TimeZoneOffset` field. The data for this specific test, though, is non-`nil`, specifically set in call to `dataTypesTest.RandomBase()`.